includes/definitions/discovery/ironware.yaml

```diff
@@ -24,6 +24,7 @@ modules:
         hardware_mib: FOUNDRY-SN-ROOT-MIB
         sysDescr_regex: '/IronWare Version V(?<version>.*) Compiled on/'
         serial: FOUNDRY-SN-AGENT-MIB::snChasSerNum.0
+        version: .1.3.6.1.4.1.1991.1.1.2.1.11.0
```

Comment:
Doesn't it have a mib name?

Reply:
`FOUNDRY-SN-AGENT-MIB::snAgImgVer.0` ?

Reply:
mixing numeric and non-numeric is bad, causes LibreNMS do snmptranslates on everything.

Reply:
Thanks, just updated to named OID.